docs/views/react.md

```diff
@@ -0,0 +1,93 @@
+# Using React or any other framework instead of HTML Templates
+
+In case of using React, Vue or any other framework, it's recommended to run [bin/prepare_other_frontend](../../bin/prepare_other_frontend.sh) script.
+
+It automatically clean ups the views (welcome page only) and sets up basic index.html into public folder. It also creates `frontend` which should be used for the React, Vue or any other frontend framework.
+
+When running `yarn build` or `npm run build` - you must move all the static files to the `public` folder.
+
+## Example (Development)
+
+In case of development you have to run go and frontend separately. You can also use docker-compose to do so.
+
+But beware, that in case of docker-compose you'll not have live reload for the golang application.
+
+### Docker Compose (Hot Reload Not Enabled in Go)
+
+1. Configure .env
+
+```
+cp .env.example .env
+```
+
+2. Run docker compose
+
+```
+docker-compose up
+```
+
+3. Navigate to `http://localhost:8000` for the Go App
+
+4. Navigate to localhost:5173 for React App
```

Comment:
I guess in step 4 could be done better as done in step 3 with the localhost link?
```suggestion
3. Navigate to `http://localhost:8000` for the Go App

4. Navigate to `http://localhost:5173` for React App
```